Resources/Prototypes/SimpleStation14/Species/ipc.yml

```diff
@@ -28,6 +28,11 @@
   id: MobIPCSprites
   sprites:
     Head: MobIPCHead
+    Snout: MobHumanoidAnyMarking
+    HeadTop: MobHumanoidAnyMarking
+    HeadSide: MobHumanoidAnyMarking
+    Wings: MobHumanoidAnyMarking
+    Tail: MobHumanoidAnyMarking
```

Comment:
So this essentially gives them the 'slots' for these markings, but without the white list set they can only be forced on. I mean like, sure I guess, seems rather unneeded to me.
One concern I have is will this cause these marking categories to be selectable in the marking editor even though they're all empty?

Reply:
The player-facing marking editor is unchanged, this just *allows* the markings to be *added*. If you want the players to, in customization, add them you'd need to edit their marking points prototype or whatever it is

Reply:
Sounds good.